### ISSUE TITLE
【Bug?】Subquery "NOT EXISTS " in WHERE clause does not work

### ISSUE THREAD

**Issue by hiroyuki0415** [MarkMpn/Sql4Cds#331]
```sql
SELECT s.*
FROM   systemuser AS s
WHERE  NOT EXISTS (SELECT *
                   FROM   businessunit AS b
                   WHERE  b.businessunitid = s.businessunitid);
```

The result of this SQL should be empty (In fact, it is empty in AzureDataStudio). 
However, this is not the case.

Thanks in advance for your support.

**Comment by MarkMpn** [comment]
I believe this is fixed in v7.4 as part of one of the other fixes for sub query errors, can you test again with the latest version please?

**Comment by hiroyuki0415** [comment]
This bug was fixed in v7.4. 
Thank you very much!😊
